TranslationNotifierGenerator/TranslationNotifierGenerator.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Any reason for not targetting .NET7?

Reply:
netstandard2.0 is required by source generator